qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -91,6 +91,11 @@ def num_qubits(self) -> int:
         """
         return self.generators.num_qubits
 
+    @property
+    def settings(self) -> dict:
+        """Return settings."""
+        return {"generators": self.generators, "rates": self.rates}
+
```

Comment:
I realize this may be out of your (our) hands, but this attribute just looks like namespace pollution to me. It's not obvious to me how these things should be considered "settings", that this attribute should be public, etc. 

For example ``__json__()`` or ``__state__()`` would be nicer to me.

I'm more complaining about the state of affairs of our ser/de rather than trying to block this PR.

Reply:
+1. It's `settings` in `quantum_info` due to historical reasons, but we are free to use a different method.